examples/demo.ckan.org.js

```diff
@@ -0,0 +1,20 @@
+const ckan = require('../lib/index')
+
+const client = new ckan.Client(
+  '8e87c120-fff8-4b5b-bd61-a66fd8b954bf', //API Key
```

Comment:
I would not commit a *real* API key. would have put XXX and people insert their own.

Reply:
But not end of world as on demo.ckan.org

Reply:
Who cares 😅 but yea agree